tensorflow/core/platform/cloud/retrying_file_system.cc

```diff
@@ -40,20 +42,22 @@ bool IsRetriable(Status status) {
 Status CallWithRetries(const std::function<Status()>& f) {
   int attempts = 0;
   while (true) {
+    int delay = std::min(initial_delay_seconds << attempts, kMaximumBackoffSeconds);
```

Comment:
Regarding 
1. I will add the `constexpr` for delay.

2. I originally added this line here as attempts is incremented at the beginning of the loop. Adding near line 53 would have a slightly uglier way of doing this -
`int delay = std::min(initial_delay_seconds << attempts - 1, kMaximumBackoffSeconds);`

Do you propose to make this change? Or should we increment at the end of the loop and initialize `kMaxAttempts` to 3?
```
constexpr int kMaxAttempts = 3;
...

Status CallWithRetries(const std::function<Status()>& f) {
  int attempts = 0;
  while (true) {
    auto status = f();
    if (!IsRetriable(status) || attempts >= kMaxAttempts) {
      return status;
    }
    LOG(ERROR) << "The operation resulted in an error and will be retried: "
               << status.ToString();
    constexpr int delay = std::min(initial_delay_seconds << attempts, kMaximumBackoffSeconds);
    attempts++;
    sleep(delay);
  }
}
```